h2/src/test/org/h2/test/db/LuceneCustomAnalyzerTest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2004-2018 H2 Group. Multiple-Licensed under the MPL 2.0,
+ * and the EPL 1.0 (http://h2database.com/html/license.html).
+ * Initial Developer: H2 Group
+ */
+package org.h2.test.db;
+
+import org.h2.test.TestBase;
+import org.h2.test.TestDb;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class LuceneCustomAnalyzerTest extends TestDb {
+
+    private Connection conn;
+
+    public static void main(String... a) throws Exception {
+        TestBase.createCaller().init().test();
+    }
+
+    private void setup() throws SQLException {
+        System.setProperty("h2.fullText.luceneAnalyzer", "org.apache.lucene.analysis.WhitespaceAnalyzer");
+        conn = getConnection("lucene-custom-analyzer");
```

Comment:
This database should be deleted somewhere before and after execution of this test case. Take a look on other test cases. `DROP ALL OBJECTS` is not enough (and is not needed if database will be deleted).

You need to delete database before tests because it may be damaged due to abnormal termination of previous execution, for example.

You need to delete database after test to avoid leaving a garbage on disk.